modules/cudaarithm/test/test_element_operations.cpp

```diff
@@ -2809,6 +2850,97 @@ INSTANTIATE_TEST_CASE_P(CUDA_Arithm, CartToPolar, testing::Combine(
     testing::Values(AngleInDegrees(false), AngleInDegrees(true)),
     WHOLE_SUBMAT));
 
+PARAM_TEST_CASE(CartToPolarInterleaved1, cv::cuda::DeviceInfo, cv::Size, AngleInDegrees, UseRoi)
```

Comment:
Can you give these test cases more informative names.  Worst case scenario you could use `CartToPolarInputInterleaved`, `CartToPolarInputOutputInterleaved`, `PolarToCartOutputInterleaved`, `PolarToCartInputOutputInterleaved` if you can't think of anything better.